src/measure_theory/lp_space.lean

```diff
@@ -1571,18 +1575,12 @@ def comp_Lpₗ (L : E →L[ℝ] F) : (Lp E p μ) →ₗ[ℝ] (Lp F p μ) :=
     simp only [ha1, ha2, ha3, ha4, map_smul, pi.smul_apply],
   end }
 
-variables {μ p}
-lemma norm_comp_Lp_le (L : E →L[ℝ] F) (f : Lp E p μ)  : ∥L.comp_Lp f∥ ≤ ∥L∥ * ∥f∥ :=
-lipschitz_with.norm_comp_Lp_le _ _ _
-
-variables (μ p)
-
-/-- Composing `f : Lp E p μ` with `L : E →L[ℝ] F`, seen as a continuous `ℝ`-linear map on
+/-- Composing `f : Lp E p μ` with `L : E →L[𝕜] F`, seen as a continuous `𝕜`-linear map on
 `Lp E p μ`. -/
-def comp_LpL [fact (1 ≤ p)] (L : E →L[ℝ] F) : (Lp E p μ) →L[ℝ] (Lp F p μ) :=
+def comp_LpL [fact (1 ≤ p)] (L : E →L[𝕜] F) : (Lp E p μ) →L[𝕜] (Lp F p μ) :=
```

Comment:
Constructions like this are often hard to find in mathlib, because you don't think about them until you need them.  While you're working here, would you mind adding a cross-reference to the similar constructions `continuous_linear_map.comp_left_continuous`, `continuous_linear_map.comp_left_continuous_bounded`, just to increase visibility?

Reply:
Done!
Doing so, I realized that I did not know what `comp_LpL` was. It was in the same section as a lemma I wanted to generalize, so it got the same treatment, but I did not even read it.